Common/models/TrendingTerm.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  TrendingTerm.swift
+//  Canillitapp
+//
+//  Created by Marcos Griselli on 08/07/2018.
+//  Copyright © 2018 Ezequiel Becerra. All rights reserved.
+//
+
+import Foundation
+import SwiftyJSON
+
+struct TrendingTerm {
+    let criteria: String
+    let quantity: Int
+    
+    init?(json: JSON) {
+        guard let criteria = json["criteria"].string,
+            let quantity = json["quantity"].int else {
+                return nil
+        }
+        self.criteria = criteria
+        self.quantity = quantity
+    }
+}
+
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Canillitapp/headlines-iOS/blob/921c8213b5513874c7a0c80f3a4d35cf853f3496/Common/models/TrendingTerm.swift#L25"></span>Files should have a single trailing newline.</td>
    </tr>
  </tbody>
</table>